pkg/resource/slo/delete.go

```diff
@@ -134,7 +134,13 @@ func (d Deleter) DeleteAll(ctx context.Context) error {
 			errs = append(errs, err)
 		}
 	}
-	return errors.Join(errs...)
+
+	retErr := errors.Join(errs...)
+	if retErr != nil {
+		log.ErrorContext(ctx, "Failed to delete all %s configurations: %v", config.ServiceLevelObjective{}, retErr)
```

Comment:
At the moment, the error log is a bit inconsistent across all resources. Sometimes the resource type starts with an uppercase letter and sometimes it's the string of the config type (lowercase)
Could you align them?